src/formatDate.js

```diff
@@ -51,6 +51,36 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const parsedDate = date.split(fromFormat[3]);
+  const newDate = [];
+  const newSeparator = toFormat[3];
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    let posInNewFormat = toFormat.indexOf(fromFormat[i]);
+
+    if (posInNewFormat !== -1) {
+      newDate[posInNewFormat] = parsedDate[i];
+
+      continue;
```

Comment:
do you really need `continue` there?)

Reply:
Yes, I do, because in that if condition I treat with the situation when indexOf has found the needed element in the toFormat array and I don't want the algorithm to deal with the part of the code where the exclusion situation is handled